west.yml

```diff
@@ -88,7 +88,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: 78efc4ba7c1057c1cf2bf06e3e27ed7cc33e1da7
+      revision: pull/87/head
```

Comment:
Please separate this change to west.yml into a separate commit.

Reply:
You will need to do this because the flow is to: 

1) have this PR point to the PR in the HAL. 
2) Have the HAL PR approved and then merged.
3) Update this PR to now point a a revision SHA

Reply:
Make them separate https://github.com/zephyrproject-rtos/zephyr/pull/34078/commits/75bb9bc8fb925b08c13c5f3f9c450d65a3846626